packages/@adobe/spectrum-css-temp/components/tags/skin.css

```diff
@@ -155,3 +155,50 @@ governing permissions and limitations under the License.
     }
   }
 }
+
+@media (forced-colors: active) {
+  .spectrum-Tags-item {
```

Comment:
Looks like the Tags are missing a distinguishable active style in WHCM. Clicking on the Tag in the "icon, removable" story in non-WHCM displays a style change but not in WHCM